resources/views/admin/posts/edit.blade.php

```diff
@@ -0,0 +1,110 @@
+                <div class="p-6 bg-white border-b border-gray-200">
+
+
+                    <div class="w-1/2 mx-auto">
+                        <x-auth-validation-errors/>
+                        <form method="POST" action="{{ route('posts.update', $post) }}" enctype="multipart/form-data">
+                            @csrf
+                            @method('PUT')
+
+                            <div>
+                                <x-label for="title" :value="__('Title')"/>
+
+                                <x-input id="title" class="block mt-1 w-full" type="text" name="title"
+                                         :value="old('title', $post->title)" required
+                                         autofocus/>
+                            </div>
+
+                            <div>
+                                <x-label for="slug" :value="__('Slug')"/>
+
+                                <x-input id="slug" class="block mt-1 w-full" type="text" name="slug"
+                                         :value="old('slug', $post->slug)" required
+                                         autofocus/>
+                            </div>
+
+                            <div>
+                                <x-label for="image" :value="__('Image')"/>
+
+                                <div class="items-center flex">
+                                    <x-input type="file"
+                                             id="image"
+                                             class="block mt-1 w-full flex-1"
+                                             name="image"
+                                             :value="old('image', $post->image)"
+                                             autofocus/>
+
+                                    <img src="{{asset('storage/' . $post->image)}}" alt="" class="sm:rounded-lg"
+                                         width="100">
+                                </div>
+
+                            </div>
+
+
+                            <div>
+                                <x-label for="excerpt" :value="__('Excerpt')"/>
+
+                                <x-form.text-area id="excerpt" name="excerpt"
+                                                  class="w-full">{{ old('excerpt', $post->excerpt) }}</x-form.text-area>
+
+                            </div>
+
+                            <div>
+                                <x-label for="body" :value="__('Body')"/>
+
+                                <x-form.text-area id="body" name="body"
+                                                  class="w-full">{{ old('body', $post->body) }}</x-form.text-area>
+
+                            </div>
+
+
+                            <div>
+                                <x-label for="category_id" :value="__('Category')"/>
+
+                                <select name="category_id" id="category_id">
+                                    @foreach(\App\Models\Category::all() as $category)
```

Comment:
I would say better would be to pass `$categories` from controller and not do it directly in views.